includes/derivatives.inc

```diff
@@ -486,3 +485,28 @@ function islandora_ocr_set_generate_ocr_rels_ext_statement(AbstractObject $objec
   $generate_ocr = $generate_ocr ? 'TRUE' : 'FALSE';
   $object->relationships->add(ISLANDORA_RELS_EXT_URI, 'generate_ocr', $generate_ocr, RELS_TYPE_PLAIN_LITERAL);
 }
+
+/**
+ * Remove the generate_hoct field if present in the RELS-EXT.
+ */
+function islandora_ocr_remove_generate_hocr_rels_ext_statement(AbstractObject $object) {
+  if ($object->relationships->get(ISLANDORA_RELS_EXT_URI, 'generate_hocr', '', RELS_TYPE_PLAIN_LITERAL)) {
+    $object->relationships->remove(ISLANDORA_RELS_EXT_URI, 'generate_hocr', '', RELS_TYPE_PLAIN_LITERAL);
```

Comment:
Also, just noticed... It was doing the current thing previously, but this should probably use `NULL` to match all values, since the empty string _is_ a valid RDF plain literal value... Of course, to make use of 'em, [Tuque itself would have to be made to differentiate between `FALSE`-y values](https://github.com/Islandora/tuque/blob/1.x/FedoraRelationships.php#L397).